lib/cli/cli.js

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node
```

Comment:
[`n/hashbang`](https://github.com/eslint-community/eslint-plugin-n/blob/master/docs/rules/hashbang.md): this hashbang comment is unnecessary.